runtime/pendulum/src/lib.rs

```diff
@@ -365,18 +408,42 @@ parameter_types! {
 	pub const MaxReserves: u32 = 50;
 }
 
+pub struct MoveDustToTreasury;
+
+impl
+	OnUnbalanced<
+		Credit<<Runtime as frame_system::Config>::AccountId, pallet_balances::Pallet<Runtime>>,
+	> for MoveDustToTreasury
+{
+	fn on_nonzero_unbalanced(
+		amount: Credit<
+			<Runtime as frame_system::Config>::AccountId,
+			pallet_balances::Pallet<Runtime>,
+		>,
+	) {
+		let _ = <Balances as Currency<AccountId>>::deposit_creating(
+			&TreasuryPalletId::get().into_account_truncating(),
+			amount.peek(),
+		);
+	}
+}
+
 impl pallet_balances::Config for Runtime {
 	type MaxLocks = MaxLocks;
 	/// The type for recording an account's balance.
 	type Balance = Balance;
 	/// The ubiquitous event type.
 	type RuntimeEvent = RuntimeEvent;
-	type DustRemoval = Treasury;
+	type DustRemoval = MoveDustToTreasury;
 	type ExistentialDeposit = ExistentialDeposit;
 	type AccountStore = System;
 	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
 	type MaxReserves = MaxReserves;
 	type ReserveIdentifier = ReserveIdentifier;
+	type FreezeIdentifier = ();
+	type MaxFreezes = ();
+	type MaxHolds = ConstU32<1>;
```

Comment:
Is there a reason why we don't just set `type MaxFreezes = ConstU32<1>;` or similar as well?
I suppose you chose `type MaxHolds = ConstU32<1>;` arbitrarily or is there a reason for this exact value?

Reply:
This is probably some of the configuration we need to agree on, but the `Get<u32>` default implementation for `()` is `0`, which would I think disallow the functionality. `MaxHolds` was also chosen arbitrarily.

Reply:
Ah okay, I didn't know that `Get<u32>` has an implementation for `()` but it makes sense. I think I'd prefer making this explicit with `ConstU32<0>` but it doesn't matter much.